blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,67 @@
+
+## Automatic Node OS Security Patching mechanisms at AKS
+
+AKS provides two managed and tested mechanisms to deliver the latest security packages to your Node Operating System. 
+
+**Automatic Node Image Channel** - AKS updates nodes weekly with a newly patched VHD for security and bug fixes. This update follows maintenance windows and upgrade configuration settings. Automatic Node image upgrades are supported as long as the cluster's Kubernetes minor version is in support. These AKS-tested node images are fully managed and applied with safe deployment practices.
+
+**OS Security Patch Channel** - Several customers may need only the security packages for their OS without additional bug fixes and updates. The OS Security Patch channel provides a fully managed, attended Node OS security-only solution. The Security-Patch channel reimages nodes only when necessary and provides live security patching updates with zero disruption, respecting planned maintenance windows and follows azure safe deployment practices.
+
+
+## When to use OS Security Patch or Automatic Node Image Channel?
+
+Choosing between the OS Security Patch channel and the Automatic Node Image channel depends on your specific requirements and operational constraints. Here's a breakdown based on common scenarios:
+
+- **Speed of Patching is Paramount**: If addressing critical vulnerabilities (CVEs) promptly is a priority, the OS Security Patch channel is the better choice. It delivers updates 1 to 2 weeks faster than the Node Image channel, ensuring your workloads remain secure against emerging threats.
+
+- **Require Comprehensive Security Fixes and Bug Fixes**: For environments where both security patches and additional bug fixes or binaries are essential, the Node Image channel is ideal. It provides a more comprehensive update approach, ensuring both security and functionality improvements.
+
+- **Workload Sensitive to Multiple Reimages in a Month**: If your workloads cannot tolerate frequent disruptions, the OS Security Patch channel is preferable. It minimizes disruptions by focusing solely on security packages, reimaging nodes 60-70% less frequently, and performing live security patching during other times.
+
+- **Using Windows Environment for Running Workloads**: Currently, the OS Security Patch channel is better suited for Linux-based workloads. For Windows environments, the Node Image channel is recommended until OS Security Patch support for Windows is introduced.
+
+- **Operating in a Capacity-Constrained Region or SKU**: In regions or SKUs with limited capacity, the OS Security Patch channel is advantageous. Its reduced frequency of node reimages helps mitigate surging and ensures smoother operations in constrained environments.
```

Comment:
Reimage doesn't require capacity. Don't confuse the user, you're thinking of Node image upgrade with surge settings, which would try to create new nodes (you need to explain this). 
We also have max Unnavailable now too